src/db/schema.ts

```
@@ -60,3 +66,75 @@ export const verification = sqliteTable("verification", {
+`;
+export const resourceUpvotes = sqliteTable(
+  "resource_upvotes",
+  {
+    resourceId: integer("resource_id")
+      .notNull()
+      .references(() => resources.id, { onDelete: "cascade" }),
+    userId: text("user_id")
+      .notNull()
+      .references(() => user.id, { onDelete: "cascade" }), // Storing user ID to prevent duplicate upvotes
+  },
+  (t) => ({
```

ESLint

[error] Unexpected parentheses around single function argument having a body with no curly braces.

(style/arrow-parens)